mpas_analysis/ocean/conservation.py

```diff
@@ -469,8 +469,10 @@ def _make_plot(self, plot_type):
 
         lineWidths = [3 for i in fields]
         if config.has_option('timeSeries', 'movingAveragePoints'):
-            movingAveragePoints = config.getint('timeSeries',
-                                                'movingAveragePoints')
+            # We assume here that movingAveragePoints is given in months
+            # and conservation output has daily frequency
+            movingAveragePoints = 30 * \
+                config.getint('timeSeries', 'movingAveragePoints')
```

Comment:
Could this jut become:
```suggestion
            movingAveragePoints = \
                config.getint('conservation', 'movingAveragePoints')
```
and then set that to 365?

Reply:
We have something like that for other situations, e.g.:
https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/mpas_analysis/default.cfg#L613

Reply:
I'm confused about where `[timeSeries]/movingAveragePoints` even gets set.  I can't find it in `default.cfg` or anywhere in the code.

Reply:
So I could be wrong but I think the moving average isn't happening at all:
https://github.com/MPAS-Dev/MPAS-Analysis/blob/89fe7495d78cc0ea30b48e7cf5b2c72c5feae4c1/mpas_analysis/ocean/conservation.py#L471-L475
I think the else clause always gets triggered.

Reply:
Anyway, @cbegeman, if you don't have time to look into this further, I'll see what I can do.

Reply:
Thanks for pointing it out. I just added a moving average option to the time series conservation task by default so we should be fine now. It ran fine and produced the averaging.